### DISCUSSION TITLE
Cannot play remote files

### DISCUSSION THREAD

**Discussion by sadikyalcin** [TheWidlarzGroup/react-native-video#4311] [Q&A]
### What are you trying to do?

I cannot play remote video and audio files using this library and `react-native-track-player`. Once I download the files, I can play them locally. It isn't an encoding issue for sure. My backend is Drupal 10.

Any one have any ideas what it could be? Or at least something I could try to get more logs?

### What have you tried?

I've turned off CSP and `Content-Disposition` header but still can't get them to play.

Below is a sample response.

### Relevant code

```
{"error": {"code": -1, "domain": "NSURLErrorDomain", "localizedDescription": "unknown error", "localizedFailureReason": "", "localizedRecoverySuggestion": ""}, "target": 2985}
```

```
HTTP/1.1 200 OK
Date: Thu, 28 Nov 2024 15:38:07 GMT
Server: Apache
Cache-Control: private
Content-language: en
X-Content-Type-Options: nosniff
X-Frame-Options: SAMEORIGIN
Expires: Sun, 19 Nov 1978 05:00:00 GMT
X-Generator: Drupal 10 (https://www.drupal.org)
Vary: X-Consumer-ID,Origin
Accept-Ranges: bytes
Last-Modified: Fri, 02 Aug 2024 11:25:16 GMT
Keep-Alive: timeout=5, max=100
Connection: Keep-Alive
Transfer-Encoding: chunked
Content-Type: video/mp4
```

### Platforms

iOS

**Comment by freeboub** [comment]
I think you should record a network trace to see what is really happening on network level.
No idea of the issue as is.
